packages/utilities/psammead-assets/src/svgs/mediaIcons.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import { C_EBON } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_HLF } from '@bbc/gel-foundations/spacings';
+
+// `currentColor` has been used to better reflect user colour choices in Firefox.
+const MediaIcon = styled.svg`
+  vertical-align: middle;
+  margin: 0 ${GEL_SPACING_HLF};
+  color: ${C_EBON};
+  fill: currentColor;
+`;
+
+const VideoMediaIcon = styled(MediaIcon)`
+  width: 0.75rem;
+  height: 0.75rem;
+`;
+
+const AudioMediaIcon = styled(MediaIcon)`
+  width: 0.8125rem;
+  height: 0.75rem;
+`;
+
+const PhotoMediaIcon = styled(MediaIcon)`
+  width: 1rem;
+  height: 0.8125rem;
+`;
+
+const icons = {
```

Comment:
Please include a usage example in the readme of this component.
Here's an example you can follow: https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-assets/README.md#usage-1
It'd be good to state that these media icons have very specific usage. These are sized to work well alongside specific text with typography group `GEL Minion`. e.g. in a Media Indicator component.